From 0833a8539f890234110f38474b34949ffef56d84 Mon Sep 17 00:00:00 2001
From: "xuegang.liu" <xuegang.liu@freescale.com>
Date: Fri, 11 Dec 2015 11:35:20 +0800
Subject: [PATCH 577/602] MA-7254 add xhdpi config in PRODUCT_AAPT_CONFIG

We should add xhdpi config by default, else there are some
layout related issues when set lcd_density to 320 on our
platform. Add xhdpi config will increase the system image
size if customer want reduce the image size they can remove
some unnecessary dpi config by themselves.

test case:
android.theme.cts.ThemeHostTest - testHoloThemes CTS fail
when set ro.sf.lcd_density to 320.

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 imx6/evk_6sl.mk       | 2 +-
 imx6/evk_6ul.mk       | 2 +-
 imx6/sabreauto_6q.mk  | 2 +-
 imx6/sabreauto_6sx.mk | 2 +-
 imx6/sabresd_6dq.mk   | 2 +-
 imx6/sabresd_6sx.mk   | 2 +-
 imx7/sabresd_7d.mk    | 2 +-
 7 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/imx6/evk_6sl.mk b/imx6/evk_6sl.mk
index b0d47f3..f24a9cd 100644
--- a/imx6/evk_6sl.mk
+++ b/imx6/evk_6sl.mk
@@ -34,7 +34,7 @@ DEVICE_PACKAGE_OVERLAYS := device/fsl/evk_6sl/overlay
 
 PRODUCT_CHARACTERISTICS := tablet
 
-PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi
+PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi xhdpi
 
 PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/android.hardware.audio.output.xml:system/etc/permissions/android.hardware.audio.output.xml \
diff --git a/imx6/evk_6ul.mk b/imx6/evk_6ul.mk
index ef660b7..c2afc87 100644
--- a/imx6/evk_6ul.mk
+++ b/imx6/evk_6ul.mk
@@ -34,7 +34,7 @@ DEVICE_PACKAGE_OVERLAYS := device/fsl/evk_6ul/overlay
 
 PRODUCT_CHARACTERISTICS := tablet
 
-PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi
+PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi xhdpi
 
 PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/android.hardware.audio.output.xml:system/etc/permissions/android.hardware.audio.output.xml \
diff --git a/imx6/sabreauto_6q.mk b/imx6/sabreauto_6q.mk
index 97ac8b7..ed0827c 100644
--- a/imx6/sabreauto_6q.mk
+++ b/imx6/sabreauto_6q.mk
@@ -38,7 +38,7 @@ DEVICE_PACKAGE_OVERLAYS := device/fsl/sabreauto_6q/overlay
 
 PRODUCT_CHARACTERISTICS := tablet
 
-PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi
+PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi xhdpi
 
 PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/android.hardware.audio.output.xml:system/etc/permissions/android.hardware.audio.output.xml \
diff --git a/imx6/sabreauto_6sx.mk b/imx6/sabreauto_6sx.mk
index 1cd7997..526d470 100644
--- a/imx6/sabreauto_6sx.mk
+++ b/imx6/sabreauto_6sx.mk
@@ -31,7 +31,7 @@ DEVICE_PACKAGE_OVERLAYS := device/fsl/sabreauto_6sx/overlay
 
 PRODUCT_CHARACTERISTICS := tablet
 
-PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi
+PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi xhdpi
 
 PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/android.hardware.audio.output.xml:system/etc/permissions/android.hardware.audio.output.xml \
diff --git a/imx6/sabresd_6dq.mk b/imx6/sabresd_6dq.mk
index b185962..c4f69d5 100644
--- a/imx6/sabresd_6dq.mk
+++ b/imx6/sabresd_6dq.mk
@@ -43,7 +43,7 @@ DEVICE_PACKAGE_OVERLAYS := device/fsl/sabresd_6dq/overlay
 
 PRODUCT_CHARACTERISTICS := tablet
 
-PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi
+PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi xhdpi
 
 PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/tablet_core_hardware.xml:system/etc/permissions/tablet_core_hardware.xml \
diff --git a/imx6/sabresd_6sx.mk b/imx6/sabresd_6sx.mk
index 05908f7..8787eee 100644
--- a/imx6/sabresd_6sx.mk
+++ b/imx6/sabresd_6sx.mk
@@ -30,7 +30,7 @@ DEVICE_PACKAGE_OVERLAYS := device/fsl/sabresd_6sx/overlay
 
 PRODUCT_CHARACTERISTICS := tablet
 
-PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi
+PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi xhdpi
 
 PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/android.hardware.camera.xml:system/etc/permissions/android.hardware.camera.xml \
diff --git a/imx7/sabresd_7d.mk b/imx7/sabresd_7d.mk
index 56f39f5..7b05a00 100644
--- a/imx7/sabresd_7d.mk
+++ b/imx7/sabresd_7d.mk
@@ -34,7 +34,7 @@ DEVICE_PACKAGE_OVERLAYS := device/fsl/sabresd_7d/overlay
 
 PRODUCT_CHARACTERISTICS := tablet
 
-PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi
+PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi xhdpi
 
 PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/android.hardware.audio.output.xml:system/etc/permissions/android.hardware.audio.output.xml \
-- 
1.8.0

